src/Environments/Cpm/CpmPlatform.cs

```diff
@@ -57,7 +57,10 @@ public override MemoryMap_v1 MemoryMap
 
         public override HashSet<RegisterStorage> CreateImplicitArgumentRegisters()
         {
-            return new HashSet<RegisterStorage>();
+            return new HashSet<RegisterStorage>()
+            {
+                Architecture.StackRegister,
+            };
         }
 
```

Comment:
Since this class inherits from `Platform` and `Platform` now does exactly this (return the stack register as an implicit argument), you can probably remove the method altogether.

Reply:
There was not implementation of `CreateImplicitArgumentRegisters` in `Platform` but I have created it. See e7fd812